lib/ruby_indexer/lib/ruby_indexer/rbs_indexer.rb

```diff
@@ -0,0 +1,67 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyIndexer
+  class RBSIndexer
+    extend T::Sig
+
+    class << self
+      extend T::Sig
+
+      sig { params(rbs_location: RBS::Location).returns(RubyIndexer::Location) }
+      def to_ruby_indexer_location(rbs_location)
+        RubyIndexer::Location.new(
+          rbs_location.start_line,
+          rbs_location.end_line,
+          rbs_location.start_column,
+          rbs_location.end_column,
+        )
+      end
```

Comment:
Do we need this method outside of this class? If not, it should be a private instance method.